src/System.Security.Cryptography.Algorithms/ref/System.Security.Cryptography.Algorithms.csproj

```diff
@@ -5,7 +5,6 @@
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="System.Security.Cryptography.Algorithms.cs" />
-    <Compile Include="System.Security.Cryptography.Algorithms.netcoreapp.cs" />
```

Comment:
@bartonjs, I merged them since we are only building for netcoreapp now. Please review.